ambry-tools/src/main/java/com/github/ambry/clustermap/HelixClusterWideAggregationTool.java

```diff
@@ -167,8 +172,9 @@ public static void main(String[] args) throws Exception {
               String jobId = statsType.toString().toLowerCase() + (isRecurrentWorkflow ? RECURRENT_JOB_SUFFIX
                   : ONE_TIME_JOB_SUFFIX);
               String taskId = statsType.toString().toLowerCase() + TASK_SUFFIX;
-              String aggregationCommand =
-                  String.format("%s_%s", HelixHealthReportAggregatorTask.TASK_COMMAND_PREFIX, reportName);
+              String aggregationCommand = String.format("%s_%s",
+                  config.isMysql ? MySqlReportAggregatorTask.TASK_COMMAND_PREFIX
```

Comment:
Minor: I am not entirely sure what will happen if the report is not found as this report may be PartitionClass report which is temporarily not supported by mysql.

Reply:
I won't worry too much about it, it's a cli tool. If we make a mistake, we can always revert it.